projects/demo/src/modules/components/link/index.ts

```diff
@@ -9,5 +9,11 @@ import {TuiDemo} from '@demo/utils';
     changeDetection,
 })
 export default class Page {
-    protected readonly examples = ['Basic', 'Icons', 'Variants', 'Long text'];
+    protected readonly examples = [
+        'Basic',
+        'Icons',
+        'Variants',
```

Comment:
Let's drop `Variants` example now, the new one covers it better

Reply:
fixed